README.md

```diff
@@ -316,7 +316,7 @@ curl -u "username:access_key" https://api.browserstack.com/5/worker/123456789/sc
 ```
 
 ## Get Details of all Browser Workers
-This API request returns comprehensive information of all browser workers you’ve created.
+This API request returns comprehensive information of all browser workers you’ve created. Also, it will return the status of the workers - either `queue` or `running`.
```

Comment:
Can we add more information about what comprehensive information? Also, rephrase to -
Returns comprehensive information, such as, x, y, and z, for all the browser workers you've created. It also returns the current status of the workers, which can be either 'queue' or 'running'. 

Another point is, the status should be 'queued' or 'in queue'. Can you ask the engineer to check if this can change?

Reply:
sure Shweta. I'll make the necessary edits and will get in touch with developer for clarification of point you've mentioned.

Reply:
Hi Shweta, I ran this API req in my machine. So currently there are 2 states - **queue** and **running**.

Reply:
Minoli, I think we need to edit the states as 'queued' instead of 'queue'. This is not a content change, but ask the dev if it's possible.

Reply:
yes, that makes sense. I'll check it with dev.